images/build/Dockerfile

```diff
@@ -115,6 +115,18 @@ RUN curl -sL https://git.io/n-install | bash -s -- -ny - \
  && ~/n/bin/n -d 10.15.2 \
  && mv /usr/local/n/versions/node /opt/nodejs \
  && rm -rf /usr/local/n ~/n
+RUN set -e \
+ && for ver in `ls /opt/nodejs`; do \
+        nodeModulesDir="/opt/nodejs/$ver/lib/node_modules"; \
+        npm_ver=`jq -r .version $nodeModulesDir/npm/package.json`; \
+        if [ ! "$npm_ver" = "${npm_ver#6.}" ]; then \
+            echo "Upgrading node $ver's npm version from $npm_ver to 6.9.0"; \
+            cd $nodeModulesDir; \
+            PATH="/opt/nodejs/$ver/bin:$PATH" \
```

Comment:
why do you update the PATH here? We'll end up with every node version in the PATH after this loop

Reply:
The update to the path is local to that one line only(probably the line break is misleading) and it does not affect the final state of the image.

Reply:
my bad, I did not pay attention to the line break